fishtest/fishtest/rundb.py

```diff
@@ -109,16 +103,20 @@ def new_run(self, base_tag, new_tag, num_games, tc, book, book_depth,
     if spsa is not None:
       run_args['spsa'] = spsa
 
+    tc_base = re.search('^(\d+)', tc)
+    if tc_base:
+      tc_base = float(tc_base[1])
```

Comment:
`re.search('^(\d+)', tc)` will this match also non-integer base tc ? (like "13.5+0.1") 
It might not really matter right now, but maybe it is confusing later.

Reply:
good catch, i updated the regex to match `13.5` and such. it won't work for `.5` and probably other edge cases for now